aws-xray/src/test/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSamplerTest.java

```diff
@@ -168,6 +168,27 @@ void defaultInitialSampler() {
     }
   }
 
+  // https://github.com/open-telemetry/opentelemetry-java-contrib/issues/376
+  @Test
+  void testJitterTruncation() {
+    sampler.close();
```

Comment:
Let's use the same pattern as `defaultInitialSampler()`, not operating on the test's "base sampler" and defining a local one within try/resources.